packages/components/src/components/faq/attach-page-faq-schema.ts

```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import { FC, useEffect } from 'react';
+import { usePathname } from 'next/navigation';
+
+export const AttachPageFAQSchema: FC<{ faqPages: string[] }> = ({ faqPages }) => {
+  const pathname = usePathname();
+
+  useEffect(() => {
+    const html = document.querySelector('html')!;
+    const path = pathname.replace('/graphql/hive', '/');
+
+    if (faqPages.includes(path) && !html.hasAttribute('itemscope')) {
```

Comment:
Like I mentioned [on Slack](https://northstar-e5s5760.slack.com/archives/C07DYCW4NUD/p1735997151371769), I doubt this check is needed.